cli-plugins/examples/helloworld/main.go

```diff
@@ -5,7 +5,7 @@ import (
 	"fmt"
 	"os"
 
-	"github.com/docker/cli/cli-plugins/manager"
+	"github.com/docker/cli/cli-plugins/metadata"
```

Comment:
Went a bit back-and-forth what made most sense;

- `"github.com/docker/cli/cli-plugins/plugin/metadata"`
- `"github.com/docker/cli/cli-plugins/metadata"`


In the end, decided on the last one (for now), because it's shared between both, but let me know what you think!